pkg/operator/ceph/config/keyring/admin.go

```diff
@@ -29,7 +29,6 @@ const (
 	adminKeyringTemplate = `
 [client.admin]
 	key = %s
-	auid = 0
```

Comment:
i take it this is just some clean-up? didn't expect auth changes related to crush map things

Reply:
Yes, I'm just trying to save some CI runtime :-).